pages/signup.js

```diff
@@ -78,33 +84,116 @@ export default function Signup() {
+                left: '50%',
+                transform: 'translate(-50%, -50%)',
+                width: 400,
+                bgcolor: 'background.paper',
+                borderRadius: 5,
+                boxShadow: 24,
+                p: 4,
+              }}>
+                <Typography sx={{fontSize: 28}}>Google Signup</Typography>
+                <InputLabel id="accountDropDown"></InputLabel>
+                <Select
+                  labelId='accountDropDown'
+                  id="accountDropDown"
+                  value={accountType}
+                  sx={{ height: 30, margin: 2, width: 336}}
+                  onChange={(e) => { handleSignUpFormInput(e.target.value, 'account_type'), setAccountType(e.target.value) }}
+                >
+                  <MenuItem value={'Mentor'}>Mentor</MenuItem>
+                  <MenuItem value={'Mentee'}>Mentee</MenuItem>
+                </Select>
+                <Stack direction="row" spacing={1}>
+                  <TextField size='small' variant="filled" placeholder='First name' sx={{
+                    my: .3,
+                    borderRadius: 5,
+                    width: '50%',
+                  }} required onChange={(e) => handleSignUpFormInput(e.target.value, 'firstname')}></TextField>
+                  <TextField size='small' variant="filled" placeholder='Last name' sx={{
+                    my: .3,
+                    borderRadius: 5,
+                    width: '50%',
+                  }} required onChange={(e) => handleSignUpFormInput(e.target.value, 'lastname')}></TextField>
+                </Stack>
+                <TextField size='small' variant="filled" placeholder='Location' sx={{ my: .3, width: 337}}  required onChange={(e) => handleSignUpFormInput(e.target.value, 'location')}></TextField>
+                <Button sx={{ my: 2,  }} onClick={(e) => signInWithGoogle(signupInfo)} className={classes.moduleGoogleButton} startIcon={<GoogleIcon />}>Sign in with Google</Button>
+              </FormControl>
+            </Modal>
+            <Modal open={facebookOpen} onClose={handleFacebookClose}>
+              <FormControl sx={{
+                position: 'absolute',
+                alignItems: 'center',
+                top: '50%',
+                left: '50%',
+                transform: 'translate(-50%, -50%)',
+                width: 400,
+                bgcolor: 'background.paper',
+                borderRadius: 5,
+                boxShadow: 24,
+                p: 4,
+              }}>
+                <Typography sx={{fontSize: 28}}>Facebook Signup</Typography>
+                <InputLabel id="accountDropDown"></InputLabel>
+                <Select
+                  labelId='accountDropDown'
+                  id="accountDropDown"
+                  value={accountType}
+                  sx={{ height: 30, margin: 2, width: 336}}
+                  onChange={(e) => { handleSignUpFormInput(e.target.value, 'account_type'), setAccountType(e.target.value) }}
```

Comment:
Lots of repeated code in here. Since it appears that both modals are using the same exact styling, we can add another styling file in the utils for these modals and export them out. Then in this file we can import and just do something like `<FormControl sx={someImportedStyling}>` and can do that everywhere for both these forms